evm/deploy-contract.js

```diff
@@ -100,43 +161,95 @@ async function deploy(options, chain) {
 
     const contractConfig = contracts[contractName];
     const constructorArgs = await getConstructorArgs(contractName, contracts);
-    const gasOptions = contractConfig.gasOptions || chain.gasOptions || null;
+    const gasOptions = contractConfig.gasOptions || chain.gasOptions || {};
     printInfo(`Constructor args for chain ${chain.name}`, constructorArgs);
     console.log(`Gas override for chain ${chain.name}: ${JSON.stringify(gasOptions)}`);
 
-    const salt = options.salt || contractName;
-    printInfo('Contract deployment salt', salt);
+    let salt;
```

Comment:
```suggestion
    let salt = options.salt || contractName;
```
applicable for both create2/create3